CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/Conversation.java

```diff
@@ -315,7 +319,8 @@ public void run() {
                 thread.start();
 
                 latch.await();
-            } catch (InterruptedException ex) {
+            } catch (final InterruptedException ex) {
+                LOGGER.log(Level.SEVERE, "Messages update was interrupted");
```

Comment:
See first reply comment. `Thread.currentThread.interrupt()` merely toggles the interrupt flag rather than actually interrupting the thread (that's already happened to get here) so it would get to the `return false` line